modules/deploy/pages/deployment-option/self-hosted/manual/linux-system-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+= Linux System Tuning
+:description: Learn how Redpanda applies automatic tunic to your Linux system.
+
+Redpanda includes numerous features to tune your Linux systems for optimal performance. You can trigger hardware-based optimizations through the `rpk redpanda tune` command, also called the autotuner. Software-based configurations are managed automatically through the `systemd` suite of tools.
+
+== User-triggered hardware tuning
+
+Users invoke the Redpanda autotuner using the `rpk redpanda tune` command to optimize native Linux nodes. It analyzes the hardware configuration of the worker node and sets appropriate kernel options. You can choose to set certain hardware parameters using `ulimit` or other system tools while using the autotuner to manage others.
+
+=== Using the autotuner
+
+The Redpanda autotuner has a large array of hardware tuners to choose from. You may choose to either execute all available tuners, or to apply only a subset of them. This is particularly important when you wish to maintain specific configurations and prevent Redpanda from altering these settings.
+
+Redpanda recommends running the autotuner as part of your production deployment workflows. This ensures any hardware configurations or updates to the autotuner itself are incorporated quickly and efficiently into your system.
+
+For the full list of available tuners, refer to the xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune.adoc[`rpk redpanda tune` guide]. When using Kubernetes, you should tune your system at the host node level. For more information on Kubernetes configuration and execution, refer to xref:deploy:deployment-option/self-hosted/kubernetes/k-tune-workers.adoc[Kubernetes worker node tuning].
+
+== Automated software configurations
+
+Redpanda makes use of systemd to manage operating system and software-based limits, such as memlocks, file handle limits, core limits, and certain distribution-specific configurations. These help ensure the system executes stably and efficiently. Some key examples of automated configurations include:
```

Comment:
The wording of the bullets does not really match the construct provided in the introductory statement. Can you please reword so that they work? For example, take each bullet and read after the intro text.
This one is particularly confusing, as it does not seem to describe an obvious automated config.
Some key examples of automated configurations include:
- Redpanda configures asynchronus input/output interfaces and scheduler affinity. This is particularly critical when you rely on real-time processing of data streams for your applications.